### PR TITLE
Problem: error in destroyCoreClient (fix #346)

### LINKED ISSUE #346: Problem: error in destroyCoreClient
remove AsyncTask, free memory directly

### PR DESCRIPTION
**Pull request by leejw51crypto**
remove AsyncTask in destroyCoreClient

### CODE REVIEW

**Comment by leejw51crypto** [comment]
AsyncTask is active whether editor playing state or not

**Comment by leejw51crypto** [comment]
adding more protection for garbage collection

**Comment by leejw51crypto** [comment]
check validity of `this` using TWeakObjectPtr

**Comment by leejw51crypto** [comment]
default polling time can be small value such as 10 milli-seconds, and wait in unreal engine blueprint `Delay`

File: Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp [hunk 1]
```diff
@@ -649,9 +711,16 @@ void APlayCppSdkActor::SendEip155Transaction(
     // if no address, return
     if (address.Num() == 0 || chain_id == 0)
         return;
+
+    TWeakObjectPtr<APlayCppSdkActor> WeakThis(this);
     AsyncTask(ENamedThreads::AnyHiPriThreadNormalTask, [Out, coreclient,
                                                         address, chain_id, info,
-                                                        this]() {
+                                                        WeakThis]() {
+        if (!WeakThis.IsValid()) {
```
Comment (rarcifa) [inline Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp:719]: This condition is evoked multiple times. Maybe we can create a helper funtion. Something like

Example
```cpp
handleIfValid(weakThis, [](auto obj) {
    // Code that requires a valid object
    obj->someMethod();
});
```

And then just invoke that function.

**Review by rarcifa — APPROVED** [review]
lgtm, left one comment

File: Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp (hunk 1 above)
Comment (leejw51crypto) [reply Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp:719]: ok, i'll check

**Pull request merged**